### DISCUSSION TITLE
delay tooltip display

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#5875] [Q&A] [answered]
Is it possible to delay when the tooltip displays? I would want to have the tooltip show up only after hovering on an object for ~0.2 seconds rather than instantaneously.

**Comment by Pessimistress** [comment, marked as answer]
You can't do this with the built-in `getTooltip` prop. You will need to implement your own tooltip and add a timer in the `onHover` callback, see examples here: https://github.com/visgl/deck.gl/blob/7.0-release/docs/developer-guide/interactivity.md#example-display-a-tooltip-for-hovered-object

**Reply by cornhundred** [reply]
Thanks @Pessimistress
